content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -620,6 +620,11 @@ spec:
 
 PersistentVolumes binds are exclusive, and since PersistentVolumeClaims are namespaced objects, mounting claims with "Many" modes (`ROX`, `RWX`) is only possible within one namespace.
 
+### PersistentVolumes typed HostPath
```

Comment:
nit: Is the type, hostPath or HostPath?

Reply:
in yaml it would be `hostPath` as with everything else in k8s, first letter would be small. But don't see that in docs, specially in non-yaml context.

Reply:
I'd suggest we spell it as hostPath, and apply coding style when appropriate.

Reply:
@tengqm updating it now.